integrationTests/chainSimulator/bridge/simpleEsdt_test.go

```diff
@@ -0,0 +1,446 @@
+package bridge
```

Comment:
Shouldn't we de-comment the tests from this PR? https://github.com/multiversx/mx-chain-go/pull/6466/files

Reply:
Also, very nice tests and comments :+1:

Reply:
not yet because we need to wait for rust framework and bridge contract framework upgrade...